src/ontology/cl-edit.owl

```diff
@@ -2789,7 +2789,7 @@ AnnotationAssertion(rdfs:label obo:IAO_0000115 "definition"^^xsd:string)
 
 AnnotationAssertion(rdfs:label obo:IAO_0000424 "expand expression to"^^xsd:string)
 
-# Annotation Property: obo:IAO_0000700 (preferred_root)
```

Comment:
This is a super weird change, its not an issue I think cause rdfs label is still there, but just odd.

Reply:
That is weird.  It's in a commented line so will not cause any problems. I think these are written by serialiser for human readability.  I wonder if this is a sign that the IAO term has two labels - perhaps with one coming from an import.  Anyway - Definitely not something to worry about for this PR, but might be worth investigating.